pkg/lib/configreader/errors.go

```diff
@@ -33,6 +33,7 @@ const (
 	ErrTooShort                      = "configreader.too_short"
 	ErrAlphaNumericDashUnderscore    = "configreader.alpha_numeric_dash_underscore"
 	ErrAlphaNumericDashDotUnderscore = "configreader.alpha_numeric_dash_dot_underscore"
+	ErrValidDockerImage              = "configreader.valid_docker_image"
```

Comment:
nit:`ErrInvalidDockerImage = "configreader.invalid_docker_image"` (and `ErrorInvalidDockerImage()` below)

Reply:
Done!